### PR TITLE
[AIRFLOW-XXX] Add a doc on how to add a new role in RBAC UI

### PR DESCRIPTION
**Pull request by feng-tao**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-XXX
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:
Add a doc on how to add role in new UI.

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

**Comment by feng-tao** [comment]
PTAL @kaxil @Fokko

File: docs/howto/add-new-role.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+The Admin could create a specific role which is only allowed to read / write certain DAGs. To configure a new role, go to ``Security`` tab
+and click ``List Roles`` in the new UI.
+
+.. image:: ../img/add-role.png
+.. image:: ../img/new-role.png
+
+
+The image shows a role which could only write to example_python_operator is created.
+And we could assign the given role to a given user using ``airflow users --role`` cli command.
```
Comment (jgao54) [inline docs/howto/add-new-role.rst]: nit-pick: "to a new user" instead of "to a given user" for clarity.

Comment (feng-tao) [reply docs/howto/add-new-role.rst]: thanks @jgao54 . updated.

**Comment by feng-tao** [comment]
PTAL @jgao54 , doc updated.

**Pull request merged**